packages/nova-routing/lib/store.js

```diff
@@ -1,7 +1,7 @@
 import { createStore, combineReducers, applyMiddleware, compose } from 'redux';
 // import { routerMiddleware } from 'react-router-redux'
 
-import { Reducers }from 'meteor/nova:lib';
+import { Reducers, Middleware }from 'meteor/nova:lib';
```

Comment:
Moved this as well, though the implementation of the extension points remains in `nova:lib`.